migrate_files.py

```diff
@@ -0,0 +1,98 @@
+import os
+
+from autowebcompat import utils
+
+LATEST_VERSION = 2
+
+
+def read_sequence(bug_id):
```

Comment:
This function and the next should be defined under `migrate_v1_to_v2`, as they are specific to this migration (if there's going to be a v2 to v3 in the future, these two functions will not be usable in both migrations).